microsoft-edge/webview2/concepts/versioning.md

```diff
@@ -3,89 +3,152 @@ description: Version Models used for Microsoft Edge WebView2
-ms.date: 05/06/2021
+ms.date: 07/23/2021
 ms.topic: conceptual
 ms.prod: microsoft-edge
 ms.technology: webview
 keywords: IWebView2, IWebView2WebView, webview2, webview, wpf apps, wpf, edge, ICoreWebView2, ICoreWebView2Host, browser control, edge html
 ---
-# Understand WebView2 SDK versions  
+# Understand WebView2 SDK versions
 
-New versions of the WebView2 SDK are shipped at the same general cadence as the Microsoft Edge \(Chromium\) browser, which is approximately every six weeks.  
+The WebView team recommends that you use the Canary preview channel of Microsoft Edge when you use a prerelease version of the WebView2 SDK package, and use the Evergreen WebView2 Runtime when you use a release version of the WebView2 SDK package.
 
-## Release and prerelease package  
+New versions of the WebView2 SDK are shipped at the same general cadence as the Microsoft Edge \(Chromium\) browser, which is approximately every six weeks.
 
-The WebView2 NuGet package contains both a release and pre-release package.  
+<!-- key points: -->
+To use the WebView2 APIs, 86.0.616.0 is the necessary (but not sufficient) minimum version of Microsoft Edge or the WebView2 Runtime.
+WebView2 APIs are forward-compatible; a WebView2 API works when using a WebView2 Runtime that has an equal or higher build number as the SDK build number in which the API was introduced.
 
-The **release package** is forward compatible and contains the following components.  
+For example, if an API is introduced in SDK 1.0.900.0, that API would work with Runtime 94.0.1000.0, but not with Runtime 90.0.700.0.  The build number is the third part of the four-part version number for the Webview2 SDK and for Microsoft Edge or the WebView2 Runtime.
 
-*   [Win32 C/C++ APIs][ReferenceWin32]
-*   .NET APIs:  [WPF][DotnetMicrosoftWebWebview2WpfNamespace], [WinForms][DotnetMicrosoftWebWebview2WinformsNamespace], and [Core][DotnetMicrosoftWebWebview2CoreNamespace].  
-    
-APIs in the SDK are fully supported.  
+Runtime 86.0.616.0 is a special release, because it is the General Availability release.  Any WebView2 API that was in the General Availability release (WebView2 SDK version 616) works with Runtime version 616 and later.
+
+When you use a WebView2 SDK that has a build number equal to or less than the WebView2 Runtime, every API that you have access to in that SDK works with that version of the Runtime.  When you use a WebView2 SDK that has a build number greater than the WebView2 Runtime, the newer APIs' implementations are not yet available in the Runtime, and therefore cannot work.
+
+If you use the Evergreen Runtime, you can't guarantee that a client's Runtime has been updated to be equal to or greater than your SDK build number, because the Admin might have turned off updating of the WebView2 Runtime, or the client might be offline long-term.  Therefore, it's a good practice to wrap any calls to relatively new WebView2 APIs by using `QueryInterface` or `try-catch`.
```

Comment:
```suggestion
If you use the Evergreen Runtime, there are edge cases where a client's Runtime is not the latest and therefore falls behind the SDK build number, because the Admin might have turned off updating of the WebView2 Runtime, or the client might be offline long-term.  Therefore, it's a good practice to wrap any calls to relatively new WebView2 APIs by using `QueryInterface` or `try-catch`.
```

Making it sounds scary.